.github/ISSUE_TEMPLATE/user-story.yml

```diff
@@ -33,23 +33,23 @@ body:
     attributes:
       label: Acceptance Criteria
       description: |
-        Please add the acceptance criteria using one or more "given, when, then" formulae
+        Please add the acceptance criteria that best describe the desired outcomes when this work is completed 
         
         Example:
-        Given that I am an administrator who has finished reviewing a domain application
+        - Application sends an email when analysts approve domain requests
+        - Domain application status is "approved"
+        
+        Example ("given, when, then" format):
+        Given that I am an analyst who has finished reviewing a domain application
         When I click to approve a domain application
         Then the domain provisioning process should be initiated, and the applicant should receive an email update.
-      value: |
-        Given 
-        When 
-        Then
     validations:
       required: true
   - type: textarea
     id: additional-context
     attributes:
       label: Additional Context (optional)
-      description: "Please include additional references, screenshots, documentation, etc. that are relevant"
+      description: "Please include additional references (screenshots, designs, documentation, etc.) that are relevant"
```

Comment:
Does this help clarify that this would be a good place for figma links? (https://miro.com/app/board/uXjVMF1Gsek=/?moveToWidget=3458764556565365873&cot=14)

Reply:
I'd recommend being even more specific and changing designs to _design links_

Reply:
Updated in https://github.com/cisagov/getgov/pull/694/commits/360ebab27f1e33ca4eb8a6c8f368a58afb56438b